exchanges/retry/src/retryExchange.test.ts

```diff
@@ -227,3 +228,105 @@ it(`should still retry if retryIf undefined but there is a networkError`, () =>
   expect(response).toHaveBeenCalledTimes(mockOptions.maxNumberAttempts);
   expect(result).toHaveBeenCalledTimes(1);
 });
+
+it('should allow retryWhen to return falsy value and act as replacement of retryIf', () => {
```

Comment:
Should this be `retryWith` instead of `retryWhen`

Reply:
damn it 😆 well, it's just a test